cmd/kube-controller-manager/controller-manager.go

```diff
@@ -46,10 +47,35 @@ func main() {
 	util.InitLogs()
 	defer util.FlushLogs()
 
-	verflag.PrintAndExitIfRequested()
+	//Functions to start and stop this daemon.
+	startRC := func(leaseUserInfo *ha.LeaseUser) bool {
+		leaseUserInfo.Running = true
+		if err := s.Run(pflag.CommandLine.Args()); err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			glog.Infof("EXITING NOW ! Killed")
+			os.Exit(1)
+		}
+		return true
+	}
+
+	endRC := func(leaseUserInfo *ha.LeaseUser) bool {
```

Comment:
stopRC? Also this is the controller-manager which runs five+ controllers. What does RC mean here?